container-scan/action.yaml

```diff
@@ -33,11 +33,14 @@ inputs:
     description: Target platform to build image for (eg. linux/amd64 (default), linux/arm64, etc)
     required: false
     default: linux/amd64
-
   build-args:
     required: false
     description: List of build arguments for docker build as key-value pairs (e.g., KEY=VALUE)
     default: ""
+  build-contexts:
+    required: false
+    description: List of additional build contexts (e.g., name=path)
+    default: ""
```

Comment:
not sure the empty string is a valid context for docker build? we could use the . as default if the empty string is not valid.

Reply:
nvm, I think empty string will still work as we are giving the context . in the beginning